<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meMatVolumes</class>
 <widget class="QWidget" name="meMatVolumes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_26">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Inner volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="innerVolumes"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeselectInner">
       <property name="text">
        <string>Deselect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_15">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_27">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Outer volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="outerVolumes"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeselectOuter">
       <property name="text">
        <string>Deselect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
